asset/css/markbind.css

```diff
@@ -11,6 +11,10 @@ code {
     word-break: normal;
 }
 
+mark > code {
```

Comment:
```suggestion
mark code {
```

Let's use the normal descendant selector instead, if not stuff like this wouldn't get affected:
```md
==12 *1 `1234234` 12* 2==

==12  1 `1234234` 12  2==
```

![image](https://user-images.githubusercontent.com/3646725/79998325-228cf480-84ed-11ea-86b4-b8e00e07d075.png)

because for the top one the `code` is not a child of `mark`.